api/admin/adminModel.js

```diff
@@ -3,39 +3,54 @@ const db = require('../../data/db-config');
 const getAdmins = async () => {
   return await db('admins').leftJoin(
     'profiles',
-    'admins.user_id',
-    'profiles.okta'
+    'admins.profile_id',
+    '=',
+    'profiles.profile_id'
   );
 };
 
 const findByAdminId = async (id) => {
-  return db('admins')
-    .leftJoin('profiles', 'admins.user_id', 'profiles.okta')
-    .where('admins.id', id);
-};
-
-const findByOkta = async (okta) => {
-  return db('admins')
-    .leftJoin('profiles', 'admins.user_id', 'profiles.okta')
-    .where('admins.user_id', okta);
+  return await db('admins')
+    .join('profiles', 'admins.profile_id', 'profiles.profile_id')
+    .where('admins.admin_id', id)
+    .first();
 };
 
 const addAdmin = async (admin) => {
-  return await db('admins').insert(admin).returning('*');
+  // we are makeing okta_id as payload we need to figure out away to save it from sever
+  /*payload  {
+    "email": "asdfasdf@maildrop.cc",
+    "name": "eooas User",
+    "okta_id": "00ulthapbErVUwasdt4x11",
+    "role": "admin"
+  }
+  ==> make sure to look at the adminMiddleware.js and make changes there
+  */
```

Comment:
These comments are for the future developer; they are there only to help them and know what kind of thought process I was going through.
We can remove these comments later and add them to the documentation.